docs/pages/application-access/getting-started.mdx

```diff
@@ -23,42 +41,47 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
+   
+   <Admonition type="note" title="Application Access and DNS for Self-Hosted Clusters">
```

Comment:
I would remove the `Admonition` tags and incorporate the text into the bullet above. Otherwise, this seems like special-case advice that readers can ignore, whereas, if a reader has Teleport Team/Enterprise Cloud, they will not need DNS subdomains.